content/welcome.md

```diff
@@ -7,21 +7,27 @@ imageMeta:
 featured: true
 authors:
   - rhd16
+  - onelastword
+  - missilemage
 date: Tue Jan 26 2021 17:02:59 GMT+100 (IST)
 ---
 
-Welcome to Scuffd Games, a podcast and group started by three friends who wanted to share their opinions on all things gaming! 
-We originally started out as a fantasy critic league, but quickly realized it could be fun for the world to hear our dumb opinions!
+Welcome to Scuffd Games, a podcast and group started by three friends who wanted to share their opinions on all things gaming!
+It all started out as a [fantasy critic league](https://fantasycritic.games), but quickly realized it could be fun for the world to hear our dumb opinions!
 In comes Scuffd, a place for us to share reviews, previews, and just our general thoughts on games and the games industry.
 
 ### Who are we?:
 
 ### Riley "rhd16" Donoho
-Hi! I'm currently a college student at the University of Utah studying Journalism! I've been playing games since I was born,
-and am excited to share my passion for the art of games with everyone!  1
 
-### Ryan Tablada 
- sdjhkgfasljkfldfdsahfa (put shit here)
+Hi! I'm currently a college student at the University of Utah studying Journalism!
+I've been playing games since I was born, and am excited to share my passion for the art of games with everyone!
+
+### Ryan Tablada
+
+A goofball on the internet that has some loud and mostly bad opinions on the internet.
+
+> "The Legend of Zelda: Breath of the Wild" is Just "Ok"
```

Comment:
this is illegal pls delete

Reply:
@rhd16 Ok I'll change it

```
> "Immortals: Fynex Rising" is better than "The Legend of Zelda: Breath of the Wild"
```

Reply:
somehow you made it even worse my god @rtablada